chroma_core/migrations/0016_task.py

```diff
@@ -47,22 +39,28 @@ class Migration(migrations.Migration):
                 ("fids_completed", models.BigIntegerField(default=0)),
                 ("fids_failed", models.BigIntegerField(default=0)),
                 ("data_transfered", models.BigIntegerField(default=0)),
+                ("single_runner", models.BooleanField(default=False)),
                 ("keep_failed", models.BooleanField(default=True)),
-                (
-                    "actions",
-                    django.contrib.postgres.fields.ArrayField(base_field=models.PositiveIntegerField(), size=None),
-                ),
+                ("actions", django.contrib.postgres.fields.ArrayField(base_field=models.TextField(), size=None),),
                 ("args", django.contrib.postgres.fields.jsonb.JSONField(default={})),
                 (
                     "filesystem",
                     models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to="chroma_core.ManagedFilesystem"),
                 ),
+                (
+                    "running_on",
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.SET_NULL,
+                        to="chroma_core.LustreClientMount",
```

Comment:
This should probably be a `ManagedHost`. If we delegate all actions to just be action plugins, then it's the actions plugins responsibility to efficiently schedule work.